NFIQ2/NFIQ2Algorithm/include/nfiq2/tool/nfiq2_ui_log.h

```diff
@@ -115,8 +115,8 @@ class Log {
 	 *    If the image was resampled 0 = not resampled, 1 = resampled.
 	 */
 	void printError(const std::string &name, uint8_t fingerCode,
-	    unsigned int score, const std::string &errmsg, const bool quantized,
-	    const bool resampled) const;
+	    const std::string &errscore, const std::string &errmsg,
```

Comment:
`errscore` is _always_ `"NA"` correct? Do you anticipate non-`"NA"` error scores? 

 * If yes, leave this parameter in, but move it to the end and give it a default argument of `"NA"`.
 * If no, remove this parameter and have a `static const std::string errorScore{"NA"};` within `printError()`.